### PR TITLE
[ch9689] file partner type command

### PR DESCRIPTION
**Pull request by domdinicola**
fix tests
Merge pull request #1178 from unicef/fix--baseline-target-allowed-chars

restrict baseline values to numbers
EquiTrack to core
Merge pull request #48 from unicef/update-app-sel

Updates app-selector to v2.1.9
9689 init-partner-file-type
Merge pull request #10 from unicef/simplify_docker_image

simplify docker image and reduce size
Merge pull request #511 from unicef/handle-rejections-on-loaders

Handle rejections on loaders
Merge pull request #81 from unicef/fix--firefox-tests

Fixes issues with tests failing on firefox 65

### CODE REVIEW

File: src/etools/applications/partners/management/commands/init-partner-file-type.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    def add_arguments(self, parser):
+        parser.add_argument('--schema', dest='schema')
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+
+        logger.info('Command started')
+
+        countries = Country.objects.exclude(name__iexact='global')
```
Comment (reinbach) [inline src/etools/applications/partners/management/commands/init-partner-file-type.py]: Any reason not to use `src.etools.libraries.tenant_support.utils.run_on_all_tenants`?

File: src/etools/applications/partners/management/commands/init-partner-file-type.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        if options['schema']:
+            countries = countries.filter(schema_name=options['schema'])
+
+        for country in countries:
+            connection.set_tenant(country)
+            logger.info('Initialization for %s' % country.name)
+
+            for _, name in FileType.NAME_CHOICES:
```
Comment (reinbach) [inline src/etools/applications/partners/management/commands/init-partner-file-type.py]: I think the `FileType` model records that are missing may be the `unicef_attachments.models.FileType`s

File: src/etools/applications/partners/management/commands/init-partner-file-type.py (hunk 1 above)
Comment (domdinicola) [reply src/etools/applications/partners/management/commands/init-partner-file-type.py]: great

File: src/etools/applications/partners/management/commands/init-partner-file-type.py (hunk 2 above)
Comment (domdinicola) [reply src/etools/applications/partners/management/commands/init-partner-file-type.py]: thanks for the feedback! I'll double check with @ntrncic !

**Pull request merged**